README.md

```diff
@@ -66,11 +70,27 @@ Please see [Azure IoT Edge Dev Resources](https://github.com/jonbgallant/azure-i
     - Make sure you enable Admin Access when you create the Azure Container Registry
 
 #### Automated Setup
-    
-You can also deploy the IoT Hub and Container Registry with this **Deploy to Azure** template:
+
+You can use the **Azure IoT Edge Dev Tool** to create a new IoT Hub and a new Edge device. This command will also print the corresponding connection strings 
+```
+iotedgedev azure --setup 
```

Comment:
Add --credentials here?

Reply:
The preferred way to login is the interactive mode. Not all azure accounts are allowed to perform a cli login, this needs to be allowed by the account directory. Only the interactive mode is guarantied to work. 
Since we removed the old `--interactive-login` flag, the only way to do interactive login is by not specifying the credentials, and we guide the user to do this by not specifying this flag in the documentation.
The same goes for `--update-dotenv` which is also omitted.  `--update-dotenv` for now will remove all comments from the active dot env file and will might confuse the users. 

These two flags are intended to be used in a CI/CD scenario.